versioned_docs/version-1.0/reference/bpmn-processes/bpmn-coverage.md

```diff
@@ -3,970 +3,690 @@ id: bpmn-coverage
 title: "BPMN coverage"
 ---
 
-The following BPMN elements are currently implemented in Zeebe:
+export const Highlight = ({children, color}) => (
+<span style={{ backgroundColor: color, borderRadius: '5px', color: '#fff', padding: '0.2rem', }}>
+{children}
+</span>
+);
+
+The following BPMN elements in <Highlight color="#df8a13">orange</Highlight> are supported. Click on
+an element to navigate to the documentation.
```

Comment:
This part is copied from the example on the documentation: https://docusaurus.io/docs/markdown-features/react#using-jsx-in-markdown